### DISCUSSION TITLE
RDP Device Notifications Issue

### DISCUSSION THREAD

**Discussion by gagestinson** [winsiderss/systeminformer#2199] [Questions / Answers] [answered]
I am a little confused on how to change my notifications for System Informer.

I frequently remote into my PC using RDP and every time I do that I get a bunch of notifications about new devices being detected. Every time I disconnect more notifications show up about them being removed (see image).

![image](https://github.com/user-attachments/assets/c18d449d-a1bf-4082-a730-f7ddc2ce95b1)

I do not want to get rid of all notifications or get rid of all device notifications. Is there a way to not have these happen when connecting thru RDP? If not I will still take some solutions to this problem I am having.

**Comment by jxy-s** [comment, marked as answer]
Right clicking the tray icon can disable or enable notifications entirely:
![image](https://github.com/user-attachments/assets/e0f76927-1bd2-41e5-8ac3-cf881c65cc9c)

You can also exclude or include notifications in the settings:
![image](https://github.com/user-attachments/assets/f9dc81c4-fd30-44f3-87ec-1cbbf780775f)

**Reply by gagestinson** [reply]
I added the `Remote*` filter to the device notifications, but I still get one for the Monitor. Less annoying now, but I would like for this not to happen either if possible.
![image](https://github.com/user-attachments/assets/28cd7bfb-3d66-4004-92a5-c088b6b174e9)

**Reply by jxy-s** [reply]
You can add more than one exclusion:
![image](https://github.com/user-attachments/assets/28ce9641-44e1-42d7-a12d-dab9d542b804)

**Reply by gagestinson** [reply]
Yes I know, I just do not know how this syntax works. What does the `*` symbolize? What does it do when it is placed before a word and what does it do when it is placed after a word?

**Reply by jxy-s** [reply]
Oh okay, sorry for the misunderstanding. It's a wildcard, it means "one or more characters here". So `Generic*` means match anything that starts with "Generic". If you do `*Generic*` that would match anything that has "Generic" anywhere in the string.
